.gitignore

```diff
@@ -7,4 +7,4 @@ coverage/
 .DS_Store
 .vscode
 .babel-cache
-.idea/
```

Comment:
Please revert the changes of this file.

Reply:
Done. I'm so bad with git. That makes me cry.

Reply:
recheck please, you removed one line in this file.

Reply:
Well... Yes, I removed `.idea` in a commit, then I realised that I forgot a line and removed `/_mytest/` in a second commit.
Those are 2 folders I use in my IDE.